src/Themes.scss

```diff
@@ -1382,6 +1388,17 @@ body.gray-theme {
         }
     }
 
+    .ToggleSwitch:not(.ToggleSwitch--checked).SoundOptionsToggle {
+        background-color: #B9BEC6;
```

Comment:
Just so I understand, the hover styles are meant to be the same for the gray theme, but not some of the other themes?

Reply:
Could you tell me more about what other themes you are thinking about?

Reply:
I think it's just high contrast that had me confused, as it has different hover styles for checked/not checked.   They all seem fine, it was more of a question.